### PR TITLE
Remove MapboxEvents 1.0.9 from dependency

### PR DESCRIPTION
**Pull request by ShanMa1991**
### Description
This Pr is to remove the Mapbox Events for iOS v1.0.9 from dependency brought in by #4144

### CODE REVIEW

**Review by azarovalex — APPROVED** [review]

File: Cartfile.resolved [hunk 1]
```diff
@@ -3,7 +3,7 @@ binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/Ma
 github "Quick/Nimble" "v9.2.1"
 github "Quick/Quick" "v3.1.2"
 github "mapbox/mapbox-directions-swift" "v2.7.0-rc.1"
-github "mapbox/mapbox-events-ios" "v1.0.9"
+github "mapbox/mapbox-events-ios" "v1.0.8"
 github "mapbox/turf-swift" "v2.5.0"
 github "pointfreeco/swift-snapshot-testing" "1.9.0"
-github "raphaelmor/Polyline" "v5.0.3"
+github "raphaelmor/Polyline" "v5.1.0"
```
Comment (azarovalex) [inline Cartfile.resolved:9]: Was it updated in Package.resolved?

Comment (ShanMa1991) [reply Cartfile.resolved:9]: No.

Comment (ShanMa1991) [reply Cartfile.resolved:9]: Thanks for the suggestion, it has been updated.

Comment (MaximAlien) [inline Cartfile.resolved:9]: If you plan to cherry-pick this PR in `release-v2.8` branch we'll have to remove update to Polyline (it's meant for `v2.9.0`).

Comment (ShanMa1991) [reply Cartfile.resolved:9]: Thanks for the suggestion, I'll update this one.

**Pull request merged**